pool/app/query/entity.ml

```diff
@@ -9,6 +9,10 @@ module Column = struct
   let to_sql m = snd m
   let create_list lst = lst
   let create m = m
+
+  let to_query_parts (field, _value) =
+    [ "order", Common.Message.Field.show field ]
```

Comment:
**fix**. same as for the `sort_order`, where do we get the `"order"` string from?

Reply:
This is the keyword for which "column" is sorted, right?
Could we make this consistent?

Reply:
@timohuber  ping

Reply:
use `Field.Order`

Reply:
fixed